README.md

```diff
@@ -364,6 +364,28 @@ reached. If no configuration file is found, Regal will use the default configura
 A custom configuration may be also be provided using the `--config-file`/`-c` option for `regal lint`, which when
 provided will be used to override the default configuration.
 
+### Loading Capabilities from URLs
+
+Starting with Regal version TODO, Regal can load capabilities from URLs with
+the `file`, `http`, or `https` schemes using the `capabilities.from.url` config
```

Comment:
I'd drop `file` here, unless there's someting that can do which isn't already covered by the other file loading option.

Reply:
b5cb5c5